src/service/cid-impl.js

```diff
@@ -147,10 +146,11 @@ export function getSourceOrigin(url) {
   // The /s/ is optional and signals a secure origin.
   const path = url.pathname.split('/');
   const prefix = path[1];
-  const mode = getMode();
-  // whitelist while localdev and file is in build/ or examples/
-  if (!(mode.localDev &&
-        (prefix == 'examples.build' || prefix == 'examples'))) {
+  // whitelist localhost
+  const localDev = location.hostname == 'localhost' ||
```

Comment:
I think what we discussed was to instead make `#isProxyOrigin` only return true if the assert would not fail.